crontab

```diff
@@ -1,16 +1,16 @@
-MAILTO=fanglin@uw.edu
-PYTHONPATH=/data/eventcal/live/lib/python2.7/site-packages
+MAILTO=fanglin@uw.edu,dannylin@uw.edu
```

Comment:
What would you think about replacing this with a mailman list so that we can get errors when you're away/out of office?

Reply:
Good suggestion!